src/lib/style/components/_download-cart-page.scss

```diff
@@ -0,0 +1,103 @@
+@use '../abstracts/variables' as SRC;
+@use '../abstracts/mixins' as SrcMixins;
+
+.DownloadCartPage {
+  padding-top: 20px;
+  .description {
+    max-width: 700px;
+    padding-top: 10px;
+  }
+  .tabs-container {
+    border-bottom: 4px solid #DCDCDC;
+  }
+  .titleContainer {
+    display: grid;
+    grid-template-columns: auto 195px;
+    * {
+      vertical-align: bottom;
+    }
+  }
+  .nav-tabs {
+    border-bottom: 0px;
+    padding-top: 20px;
+    display: flex;
+    flex-wrap: nowrap;
+    height: 70px;
+    align-items: center;
+    .active {
+      padding-top: 5px;
+      border-bottom: 5px solid;
+      border-bottom-color: SRC.$secondary-action-color;
+      button {
+        color: #515359;
+      }
+      .iconImg {
+        filter: invert(1)
```

Comment:
Does this work in all browsers we try to support? I think support for filter is weird

Reply:
unknown, but I think this is a copy-paste issue.  removing from here.